examples/README.md

```diff
@@ -0,0 +1,99 @@
+
+### Creating new examples
+
+Read the [Templates Guideline](./templates/README.md) for the detailed information about new code examples creation.
+
+### Editing existing examples
+
+Sometimes you want to edit existing code examples that live in the `/examples/<version_number>` directory.
+
+1. Edit code examples within the `/examples/<version_number>` directory.
+2. Commit and push the code examples to the repo.
+3. In Github Actions run a manual workflow with the `<version_number>` input of the Handsontable and Handsontable's wrapper version.
+4. The code examples will be deployed to the Github Pages and will be available under the same URL as they already were.
+
+### Development
+
+You can launch the `http-server` or any other server if you want to test and check code examples live.
+
+To see code examples in action run these commands:
+
+**Important:** As the `next` directory of the examples uses the local builds of `handsontable` and the wrappers, for the `next` build process to work, all the root-level packages need to be built (for example, bu running `npm run all build`) before running the examples build script.
+
+1. `npm examples:install <version_number>` - will install the dependencies of all the examples matching the `<version_number>`, utilizing the `examples`' internal workspace logic.
+2. `npm run examples:build <version_number>` - will build each code example in the `/examples/<version_number>` directory then copy each example's production output to the `/examples/tmp/<version_number>`. The path to the code example in the `/examples/tmp` follows the [Folder structure](#folder-structure) convention.
+3. `npm run examples:start` - it will start the `http-server` right in the `/examples/tmp` on PORT `8080`. So the URL to the specific code example would be `http://localhost:8080/8.1.0/docs/angular/custom-context-menu/`. The URL follows the same convention as mentioned in the [Live on production](#live-on-production) section.
+
+#### How to run the arbitrary code example
+
+You may want to launch an arbitrary code example instead of a whole folder with all code examples for a specific version. To do that you will need to copy the code example production output to the `/examples/tmp` directory. An example:
+
+To launch only one code example which path is `/examples/8.1.0/docs/angular/custom-context-menu` you must copy the `dist` folder from this example and paste it to the `/examples/tmp/<your_temporary_folder_name>`. After done this, you can run `npm run examples:start` and see your code example under the URL `http://localhost:8080/<your_temporary_folder_name>/`
+
+Also, each example has `npm run start` command which will start a development server. After this command is called visit http://localhost:8080.
```

Comment:
This is too complicated. There is no need to copy anything. As stated in the last sentence, you need to go to the chosen example and use the `npm run *` commands there. An `npm run build` and `npm run start` should be enough.

Reply:
https://github.com/handsontable/handsontable/pull/7588#pullrequestreview-602786111